### PR TITLE
Issue 331 - Removing the anchor tag from the maintenance page

### PR DESCRIPTION
**Pull request by robbiethegeek**
Removing in both the system maintenance.tpl.php and the bartik theme's maintenance.tpl.php.

### CODE REVIEW

File: core/themes/bartik/templates/maintenance-page.tpl.php [hunk 1]
```diff
@@ -33,7 +33,7 @@
           <?php if ($site_name): ?>
             <div id="site-name"<?php if ($hide_site_name) { print ' class="element-invisible"'; } ?>>
               <strong>
-                <a href="<?php print $front_page; ?>" title="<?php print t('Home'); ?>" rel="home"><span><?php print $site_name; ?></span></a>
+                <spam><?php print $site_name; ?></span>
```
Comment (quicksketch) [inline core/themes/bartik/templates/maintenance-page.tpl.php]: Should probably be "span", not "spam" ;)

What's the purpose of these span tags anyway? Can we just nix them?

**Comment by robbiethegeek** [comment]
HAHA wow that is the best fat finger I have seen in a while and I did it :)

**Comment by quicksketch** [comment]
Super!

**Comment by quicksketch** [comment]
Oh! Dang merged it without checking the commit message. In the future. Please format your commit messages as: "Issue #xx: Description of issue." so we can reference these back to the original from Git blame. Thanks!

**Pull request merged**